docs/_source/_common/snippets/start_page.md

```diff
@@ -1,36 +1,78 @@
-::::{tab-set}
+Welcome to Argilla! 👋😁 We're thrilled to have you on board. Beginning with Argilla is as simple as it gets, and you've successfully taken the first step!
 
-:::{tab-item} Feedback datasets
+To start using Argilla, you need to create your first dataset. Let's get started. 🚀
 
-```python
-# install datasets library with pip install datasets
-import argilla as rg
-from datasets import load_dataset
+[For in-depth user guides, check the documentation 📖.](https://docs.argilla.io/en/latest/)
+
+# 1. Open an IDE, Jupyter or Collab
+
+Open your favorite IDE, Jupyter or Collab. If you're a Collab user, you can directly use our [introductory tutorial](https://colab.research.google.com/github/argilla-io/argilla/blob/develop/docs/_source/getting_started/quickstart_workflow_feedback.ipynb).
+
+# 2. Install the SDK with pip
 
-# load an Argilla Feedback Dataset from the Hugging Face Hub
-# look for other datasets at https://huggingface.co/datasets?other=argilla
-dataset = rg.FeedbackDataset.from_huggingface("argilla/oasst_response_quality", split="train")
+To upload and read datasets from Argilla, you need to use the Argilla SDK. You can install the SDK with pip as follows:
 
-# push the dataset to Argilla
-dataset.push_to_argilla("oasst_response_quality")
+```sh
+pip install argilla
 ```
-:::
 
-:::{tab-item} Other datasets
+# 3. Connect to your Argilla server
+
+1. Get your `API_URL`:
+
+   * If you are using Docker, it is the URL shown in your browser (by default `http://localhost:6900`)
+   * If you are using HF Spaces, it should be constructed as follows: `https://[your-owner-name]-[your_space_name].hf.space`
+
+2. Get your `API_KEY` by clicking on your user icon in the top right corner of the Argilla UI, selecting "My settings" and copying the API key.
+
+3. Connect the SDK with the server using `rg.init`. Make sure to replace "<your-api-url>" and "<your-api-key>" with the values you obtained in the previous steps.
 
 ```python
-# install datasets library with pip install datasets
 import argilla as rg
-from datasets import load_dataset
 
-# load dataset from the hub
-dataset = load_dataset("argilla/gutenberg_spacy-ner", split="train")
+rg.init(
+    api_url="<your-api-url>",
+    api_key="<your-api-key>"
+)
+```
 
-# read in dataset, assuming its a dataset for token classification
-dataset_rg = rg.read_datasets(dataset, task="TokenClassification")
+# 4. Create your first dataset
 
-# log the dataset
-rg.log(dataset_rg, "gutenberg_spacy-ner")
+1. Specify a workspace where the dataset is to be saved. You can check your workspaces in the Argilla UI by clicking on "My settings".
+
+2. Create a FeedbackDataset with two labels ("sadness" and "joy"). Don't forget to replace "<your-workspace>" with the workspace you want to use:
```

Comment:
Maybe reference the general `create_dataset`page?

Reply:
And in the reference I would mention that Argilla is fully configurable to your needs but that we will be using one of our templates.